packages/react-components/src/components/ChatMessageComponent.tsx

```diff
@@ -162,6 +163,7 @@ export const ChatMessageComponent = (props: ChatMessageProps): JSX.Element => {
             : undefined}
         </Text>
       }
+      details={message.payload.editedOn ? <div className={chatMessageEditedTagStyle(theme)}>Edited</div> : undefined}
```

Comment:
@mgamis-msft  what files do we need to add this "editted" string to for it to be using our en_us locale?

Reply:
Hmm, interesting question, ideally this is still under messageThread component(this component is something for code separation), we can still put it there for external customers

Reply:
Will send another PR for localization of the whole editing feature once it is all finished